providers/sgx/sgx-runner/src/main.rs

```diff
@@ -68,32 +65,41 @@ enum SubCommand {
         key_backup_data_path: PathBuf,
         #[structopt(short)]
         recover_consensus_key: bool,
+        #[structopt(short, parse(from_occurrences))]
+        v: u32,
     },
     #[structopt(name = "start", about = "Start tmkms process")]
     /// start tmkms process
     Start {
         #[structopt(short)]
         config_path: Option<PathBuf>,
+        #[structopt(short, parse(from_occurrences))]
+        v: u32,
     },
 }
 
-fn main() {
-    let opt = TmkmsLight::from_args();
-    let log_level = match opt.v {
+fn set_log(v: u32) -> String {
+    let log_level = match v {
         0 | 1 => Level::INFO,
-        2 => Level::Debug,
+        2 => Level::DEBUG,
         _ => Level::TRACE,
     };
     let log_level_str = format!("{}", log_level);
     let subscriber = FmtSubscriber::builder().with_max_level(log_level).finish();
-
     tracing::subscriber::set_global_default(subscriber).expect("setting default subscriber failed");
-    let result = match opt.subcommand {
+    log_level_str
```

Comment:
could this directly return `Level` instead of a string?

Reply:
For https://github.com/crypto-com/tmkms-light/blob/main/providers/sgx/sgx-app/src/main.rs#L8

1. maybe just switch to `DEBUG` if the verbose flag is provided?

Reply:
something like:
```
    let log_level = match args.next() {
        Some(s) if s == "verbose" => Level::DEBUG,
        _ => tracing::Level::INFO,
    };
```

so that there's no panic?

Reply:
ok